src/SDKs/IotHub/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
+    <AzureApiTag />
```

Comment:
@montgomp why was this file created? Please undo this change.

Reply:
done

Reply:
@shahabhijeet 7 days ago you told me to remove this file. Today (2018/01/09) you tell me I need the AzSdk.RP.props file. I am unable to fulfill both requests.